backend/src/main/java/com/znane_groby/backend/configuration/LoadDatabase.java

```diff
@@ -0,0 +1,135 @@
+        Map<String, City> citiesMap = new HashMap<>();
+        Map<String, Graveyard> graveyardsMap = new HashMap<>();
+        //Data format I am parsing
+        //cemetery,cemeteryLabel,adm_location,adm_locationLabel,location
+        //http://www.wikidata.org/entity/Q8510713,Cmentarz Zasłużonych Wielkopolan,http://www.wikidata.org/entity/Q268,Poznań,Point(16.9289 52.4144)
+        try (Stream<String> stream = new BufferedReader(new InputStreamReader(inputStream)).lines()) {
+
+            stream
+                    .skip(1)
+                    .forEach(e -> {
+                        String[] fields = e.split(",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)");
+                        if (fields.length > 3) {
+
+                            String wikiName = fields[2];
+                            String name = fields[3];
+                            citiesMap.putIfAbsent(wikiName, City.builder()
+                                    .name(name)
+                                    .wikiName(wikiName)
+                                    .build());
+                        }
+                    });
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        inputStream = getClass()
+                .getClassLoader().getResourceAsStream(fileName);
+        try (Stream<String> stream = new BufferedReader(new InputStreamReader(inputStream)).lines()) {
+            stream
+                    .skip(1)
+                    .forEach(e -> {
+                        String[] fields = e.split(",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)");
+                        if (fields.length > 1 && !fields[0].isEmpty()) {
+                            String wikiName = fields[0];
+                            String name = fields[1];
+                            double longitude = 0;
+                            double latitude = 0;
+                            String cityWikiName = "";
+                            if(fields.length > 2)
+                                cityWikiName = fields[2];
+                            if(fields.length > 4) {
+                                longitude = Double.parseDouble(fields[4].replaceAll("[^0-9.\\s]", "").split("\\s")[0]);
+                                latitude = Double.parseDouble(fields[4].replaceAll("[^0-9.\\s]", "").split("\\s")[1]);
+                            }
+                                graveyardsMap.putIfAbsent(wikiName, Graveyard.builder()
+                                        .name(name)
+                                        .wikiName(wikiName)
+                                        .city(citiesMap.get(cityWikiName))
+                                        .longitude(longitude)
+                                        .latitude(latitude)
+                                        .build());
+                        }
+                    });
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+
+//        citiesMap.entrySet().stream()
```

Comment:
Zamiast wykomentowywać log używaj niższych poziomów, np. `log.debug(...)` albo nawet `log.trace(...)` a potem używaj konfiguracji springa (logging.level.com.znane_groby....: TRACE)

Reply:
hm... tylko to badziewie generuje masę printów i tylko w wyjątkowych sytuacjach chciałbym to mieć. Generalnie, są to raczej dev debug printy pod tą jedną, jedyną rzecz. I wtedy też dobrą praktyką jest to ustawiać?

Reply:
Chodzi o to, że wykomentowany kod tego typu bardzo źle wygląda. Musisz się zdecydować: albo raz miałeś problem i nigdy się go nie spodziewasz, to kasujesz to w ogóle, albo zakładasz że może się zdarzyć tu kłopot w przyszłości i ustawiasz log trace/debug, wyłączasz ten poziom logowania na codzień dla tej klasy konfiguracją, a jeśli będzie trzeba znów włączyć to sobie poprawisz znów w konfiguracji. 

A w ogóle to polecam wdrażać się w debugger - zwykle szybciej się znajduje problem niż logowaniem.